packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -75,8 +87,11 @@ const releaseActionController = {
       };
     });
 
+    const groupName = getGroupName(query.groupBy);
+    const groupedData = _.groupBy(groupName)(formattedData);
```

Comment:
Not related to the code, but I remember that the original version of the designs had the ability to group by "document", not just content type and locale.

Did you get to the conclusion that it wasn't possible?

Reply:
Currently in the designs we actually only have options for locale and contentType. I added "action" because at some point that was an option as well in the designs but we weren't sure if it would be possible or not. 

For the "document" we are using the mainField _value_  (same implementation as bulk publish). I don't think it would make sense to groupBy mainField value since no entry will have the same.... 

Maybe we misinterpreted the design but I would also find it strange to have the mainField in this column like "title" and anyway that wouldn't make sense either for the grouping because it would basically be the same as grouping by contentType...right?

I think this column will only really make sense for grouping when all dimensions of a document share some sort of unique ID.

Reply:
okay I see, yes maybe this can be added in v5